### PR TITLE
fix: Error Fix to Hydration Error On Windows Platform.

### PR DESCRIPTION
**Pull request by myagizmaktav**
# Description

**discord username: Yağız M.#0539**

**closes #issue**

## Things to check before creating a PR
- [ ] I have inspected my topic, checked.
- [ ] If it is a core feature, executed detailed tests.

## Creating PR rules
- [ ] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [ ] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [ ] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [ ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [ ] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [ ] Clean commit history: The commits where the changes are made should be clear and organized.
- [ ] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [ ] A new feature (a change that adds a new feature, not a breaking change)
- [ ] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [ ] Documentation change

# How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

**Test Configuration**:

* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,6 +30,8 @@ build/Release
 node_modules
 jspm_packages
 .hintrc
+package-lock.json
+yarn.lock
```
Comment (el) [inline .gitignore:34]: yarn.lock should not be added to the gitignore, we use yarn so adding package-lock is unnecessary. Please revert both of these

File: package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
     "react-dom": "18.2.0",
     "react-i18next": "^12.1.5",
     "react-leaflet": "^4.2.0",
+    "react-leaflet-custom-control": "^1.3.3",
```
Comment (el) [inline package.json:52]: This should be removed

File: yarn.lock [hunk 3]
```diff
@@ -4661,6 +4661,11 @@ react-is@^18.2.0:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-18.2.0.tgz#199431eeaaa2e09f86427efbb4f1473edb47609b"
   integrity sha512-xWGDIW6x921xtzPkhiULtthJHoJvBbF3q26fzloPCK0hsvxtPVelvftw3zjbHWSkR2km9Z+4uxbDDK/6Zw9B8w==
 
+react-leaflet-custom-control@^1.3.3:
```
Comment (el) [inline yarn.lock:4664]: This change should be reverted

**Review by el — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (fdemir) [reply .gitignore:34]: so true

Comment (myagizmaktav) [reply .gitignore]: fixed

File: yarn.lock (hunk 3 above)
Comment (myagizmaktav) [reply yarn.lock]: fixed

File: package.json (hunk 2 above)
Comment (myagizmaktav) [reply package.json]: fixed

**Comment by usirin** [comment]
closing this, you can reopen after build passes.

**Pull request closed without merging**